README.md

```diff
@@ -70,11 +70,21 @@ Please do not use the issue tracker for security issues.
 
 ## Setup
 
+Ensure you have Node and Yarn installed.
+
+The latest Node version officially supported by OSx and Hardhat is 16. Node >=19 also works, but is technically unsupported by Hardhat. Use [nvm](https://github.com/nvm-sh/nvm) to switch:
```

Comment:
Here we would suggest a node version manager such as NVM instead of straight forcing it on the user